torchreid/utils/reidtools.py

```diff
@@ -77,15 +77,16 @@ def _cp_img_to(src, dst, rank, prefix, matched=False):
         qimg_path, qpid, qcamid = query[q_idx]
         num_cols = topk + 1
         grid_img = 255 * np.ones((height, num_cols*width+topk*GRID_SPACING+QUERY_EXTRA_SPACING, 3), dtype=np.uint8)
-
+        qimg_path_name = qimg_path[0] if isinstance(qimg_path, tuple) or isinstance(qimg_path, list) else qimg_path
```

Comment:
You can shorten this
```
qimg_path_name = qimg_path[0] if isinstance(qimg_path, (tuple, list)) else qimg_path
```

Reply:
Sure I just use the same logic you had above. Feel free to shorten it in another update. FYI thanks for the repo, loving how easy it is to modify it to my needs :D.